app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -173,6 +173,12 @@ def finalize
           history = ::Katello::ContentViewHistory.find(input[:history_id])
           history.status = ::Katello::ContentViewHistory::SUCCESSFUL
           history.save!
+
+          version.repositories.each do |repo|
+            SmartProxy.pulp_master.pulp_api.extensions.send(:module_default).
+              copy(repo.library_instance.pulp_id,
+              repo.pulp_id)
+          end
```

Comment:
i think this was a rebase mistake.

Reply:
:scream: thank good I still had a second working copy in my forklift :relieved: